lib/mistral_tokenizer/mistral_tokenizer.dart

```diff
@@ -170,7 +174,7 @@ class MistralTokenizer {
             //
             // if it does happen, let's follow the js implementation
             // and add <UNK> token instead of crash
-            debugPrint(
+            print(
```

Comment:
If we want to keep this print I suggest using lint ignore on it

Reply:
Oh that's a mistake. Thx for catching it. I'll change it back to debugPrint.

Reply:
Fixed in https://github.com/nomtek/flutter_ai_examples/pull/7/commits/2eea6ff4be95f983b20e512d48c0edd2c75f56b8